core-web/libs/portlets/edit-ema/portlet/src/lib/edit-ema-editor/edit-ema-editor.component.ts

```diff
@@ -140,8 +140,9 @@ export class EditEmaEditorComponent implements OnInit, OnDestroy {
             .subscribe((event: MessageEvent) => {
                 this.handlePostMessage(event)?.();
             });
-
-        this.store.updateEditorState(EDITOR_STATE.LOADING);
+        // Think is not necessary, if is Headless, it init as loading. If is VTL, init as Loaded
+        // So here is re-set to loading in Headless and prevent VTL to hide the progressbar
+        // this.store.updateEditorState(EDITOR_STATE.LOADING);
```

Comment:
It's expected that it gets stuck in loading because we are not making any interaction with the VTL for now, is okay to leave this comment to test if the VTL is indeed rendered